app/mailers/anonymous_mailer.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# AnonymousMailer handles all email sending not associated with a user. It expects to be called
+# using `with` that receives an `email` string value.
+#
+# You MUST deliver these messages using `deliver_now`. Anonymous messages rely on a plaintext email
+# address, which is personally-identifiable information (PII). All method arguments are stored in
+# the database when the email is being sent asynchronusly by ActiveJob and we must not put PII in
+# the database in plaintext.
```

Comment:
I'm currently writing up a one-pager proposal for how we might address this moving forward, so I'm hoping this is a temporary limitation.